<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PyMDFMainWindow</class>
 <widget class="QMainWindow" name="PyMDFMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>asammdf</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/asammdf.png</normaloff>:/asammdf.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>604</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/file.png</normaloff>:/file.png</iconset>
       </attribute>
       <attribute name="label">
        <string>Single files</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="files">
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="documentMode">
           <bool>false</bool>
          </property>
          <property name="tabsClosable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>782</width>
         <height>604</height>
        </rect>
       </property>
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/list.png</normaloff>:/list.png</iconset>
       </attribute>
       <attribute name="label">
        <string>Multiple files</string>
       </attribute>
       <layout class="QGridLayout" name="files_layout" rowstretch="1,0">
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,0,0,0,0">
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Compression</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cs_format"/>
          </item>
          <item row="5" column="1">
           <widget class="QComboBox" name="cs_compression"/>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="cs_split">
            <property name="text">
             <string>Split data blocks</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="cs_split_size">
            <property name="suffix">
             <string>MB</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Split block size</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="cs_btn">
            <property name="text">
             <string>Concatenate</string>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/stack.png</normaloff>:/stack.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Output format</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Operation</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="concatenate">
               <property name="text">
                <string>Concatenate</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="stack">
               <property name="text">
                <string>Stack</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="sync">
            <property name="text">
             <string>Sync using measurements timestamps</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_memory_minimum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>minimum</string>
   </property>
   <property name="toolTip">
    <string>Minimal memory usage by loading only the nedded block addresses</string>
   </property>
  </action>
  <action name="action_memory_full">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>full</string>
   </property>
   <property name="toolTip">
    <string>Load all blocks in the RAM</string>
   </property>
  </action>
  <action name="action_memory_low">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>low</string>
   </property>
   <property name="toolTip">
    <string>Load metdata block in RAM but leave the samples on disk</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
